src/data/_contributors.yaml

```diff
@@ -2022,16 +2050,4 @@ maciejzasada:
   country: UK
   role:
     - author
-  email: maciej@unit9.com
-
-skikuchi:
```

Comment:
Why is it moved up?

Reply:
I had some issues where tests verified that names were ordered alphabetically. I have put the alphabetical order back.